### PR TITLE
Implement blendmode for overlays

### PR DESCRIPTION
**Pull request by patrick-tolosa**
- Added a Utility to match ESM to default packages (expectESMToMatchDefault)
- Added implementation needed for BlendMode
- Refactored some tests to reduce bloat in the amount of lines

### CODE REVIEW

File: src/params/blendMode/BlendMode.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+/**
+ * @description Add an overlay image blended using the 'overlay' blend mode.
+ */
+function overlay():BlendMode {
+  return new BlendMode('overlay');
+}
+
+export {
+  screen,
```
Comment (maoznir) [inline src/params/blendMode/BlendMode.ts:39]: I think blend mode should include: multiply screen mask anti_removal

File: __TESTS__/TestUtils/expectESMToMatchDefault.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @description given an ESM package and a default export, this function expects the two to be identical
```
Comment (maoznir) [inline __TESTS__/TestUtils/expectESMToMatchDefault.ts:2]: should be part of a different pr?

File: src/params/blendMode/BlendMode.ts (hunk 1 above)
Comment (patrick-tolosa) [reply src/params/blendMode/BlendMode.ts:39]: This is aligned with Kotlin, 

Mask is not documented and believed to be an error in the spec
anti_removal is snoozed for later discussion

File: __TESTS__/TestUtils/expectESMToMatchDefault.ts (hunk 2 above)
Comment (patrick-tolosa) [reply __TESTS__/TestUtils/expectESMToMatchDefault.ts:2]: I had to write all this code for the tests, all I did was just put it in a different function. 
I believe it belongs here
